universal-application-tool-0.0.1/test/auth/ProfileMergeTest.java

```diff
@@ -4,265 +4,74 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import auth.oidc.IdcsProfileAdapter;
-import auth.oidc.InvalidOidcProfileException;
-import auth.saml.SamlCiviFormProfileAdapter;
-import io.ebean.DB;
-import io.ebean.Database;
 import java.util.concurrent.ExecutionException;
 import javax.inject.Provider;
-import models.Account;
 import org.junit.Before;
 import org.junit.Test;
-import org.pac4j.core.profile.definition.CommonProfileDefinition;
 import org.pac4j.oidc.profile.OidcProfile;
-import org.pac4j.saml.profile.SAML2Profile;
-import repository.ResetPostgres;
 import repository.UserRepository;
+import repository.WithPostgresContainer;
 
-public class ProfileMergeTest extends ResetPostgres {
+public class ProfileMergeTest extends WithPostgresContainer {
 
-  private IdcsProfileAdapter idcsProfileAdapter;
-  private SamlCiviFormProfileAdapter samlProfileAdapter;
+  private IdcsProfileAdapter profileAdapter;
   private ProfileFactory profileFactory;
-  private Database database;
-
-  @Before
-  public void setupDatabase() {
-    database = DB.getDefault();
-  }
 
   @Before
   public void setupFactory() {
     profileFactory = instanceOf(ProfileFactory.class);
     UserRepository repository = instanceOf(UserRepository.class);
-    idcsProfileAdapter =
+    profileAdapter =
         new IdcsProfileAdapter(
-            /* configuration = */ null,
-            /* client = */ null,
+            null,
+            null,
             profileFactory,
             new Provider<UserRepository>() {
               @Override
               public UserRepository get() {
                 return repository;
               }
             });
-    samlProfileAdapter =
-        new SamlCiviFormProfileAdapter(
-            /* configuration = */ null,
-            /* client = */ null,
-            profileFactory,
-            new Provider<UserRepository>() {
-              @Override
-              public UserRepository get() {
-                return repository;
-              }
-            });
-  }
-
-  private OidcProfile createOidcProfile(String email, String issuer, String subject) {
```

Comment:
It looks like you're deleting a lot that I added in the previous PR, is this Git oddness that needs to be fixed up or intended?

Reply:
Git incompetence :( Thank you for saving me @shubha-rajan